x11-misc/rofi-wayland/rofi-wayland-1.7.5.ebuild

```diff
@@ -0,0 +1,73 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+S=${WORKDIR}/rofi-${PV}+wayland1
+inherit meson toolchain-funcs xdg-utils
+
+DESCRIPTION="A window switcher, run dialog and dmenu replacement"
+HOMEPAGE="https://github.com/lbonn/rofi"
+SRC_URI="https://github.com/lbonn/rofi/releases/download/${PV}%2Bwayland1/rofi-${PV}+wayland1.tar.xz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="amd64 arm64 x86"
```

Comment:
GURU ebuilds are testing (and not stable)
Gentoos marks testing packages as ~arch (for example ~amd64, ~x86)

Have you tested this works on arm64? I have to look up if we even allow anything other than ~amd64 and ~x86 in the guru